tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -1823,6 +1823,13 @@ def test_orderby_user_display_with_aggregates(self):
         # because we're ordering by `user.display`, we expect the results in sorted order
         assert result == ["catherine", "cathy@example.com"]
 
+    @pytest.mark.skip(
+        """
+         For some reason ClickHouse errors when there are two of the same string literals
+         (in this case the empty string "") in a query and one is in the prewhere clause.
+         Does not affect production or ClickHouse versions > 20.4.
```

Comment:
Should we bump our test version of Clickhouse? Or are we not running > 20.4 in prod?

Reply:
We're still on 20.3 in prod so it's probably better to keep this aligned for now.